extensions/amp-ad-exit/0.1/test/test-amp-ad-exit.js

```diff
@@ -139,6 +156,13 @@ describes.realWin('amp-ad-exit', {
     win = env.win;
     toggleExperiment(win, 'amp-ad-exit', true);
     addAdDiv();
+    win.top.document.body.getResourceId = () => '6789';
```

Comment:
TODO: remove after rebase.

Reply:
Changed code to match other PR, added TODO comment

Reply:
Actually, had to undo that. Will just add TODO as recommended.